pkg/cmd/pipelinerun/list_test.go

```diff
@@ -136,6 +140,33 @@ func TestListPipelineRuns(t *testing.T) {
 		t.Errorf("unable to create dynamic client: %v", err)
 	}
 
+	tdc3 := testDynamic.Options{
+		PrependReactors: []testDynamic.PrependOpt{
+			{
+				Verb:     "list",
+				Resource: "pipelineruns",
+				Action: func(action k8stest.Action) (bool, runtime.Object, error) {
+					if action.(k8stest.ListAction).GetListRestrictions().Fields.String() == "metadata.name=pr0-1" {
```

Comment:
Extract `(k8stest.ListAction).GetListRestrictions().Fields.String()` a variable and then check.